### ISSUE TITLE
Move the toolbox file FSDA.mltbx inside /release folder

### ISSUE THREAD

**Issue by AldoCorbelliniUNIPR** [bpancras/FSDA#18]

**Comment by AldoCorbelliniUNIPR** [comment]
We used to put FSDA.mltbx inside the FSDA/bin folder of the repo but each time we added a new release of FSDA to the GitHub Repo, the sheer size of the repo grown by 50MB, until we reached 950MB.
We discussed how to debloat FSDA with Rob and Jos during our visit to MRS, and the solution was to use a BFG utility ([https://rtyley.github.io/bfg-repo-cleaner/](https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Frtyley.github.io%2Fbfg-repo-cleaner%2F&data=05%7C01%7Caldo.corbellini%40unipr.it%7C900099ff5edc4437d7d608dbed9d50c1%7Cbb064bc5b7a841ecbabed7beb3faeb1c%7C0%7C0%7C638365032143988018%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=codFqY54YVT0YXwmqNp5SGtKB6KQR9x8jAzAh%2FYvwYg%3D&reserved=0))  to remove all instances of FSDA.mltbx in the repo timeline.
 
Given that the FSDA toolbox is built by a GitHub Action in the cloud and placed in the artifact section (where is supposed to be to be linked by File Exchange), we think that this could be the best solution for us.
Anyway, let us know if you agree or have a better idea!

**Comment by bpancras** [comment]
I agree with you. Placing the MLTBX file in the artifacts section of GitHub is the right way to go.
